examples/content_management_examples/public/examples/todos/stories/todos_client.ts

```diff
@@ -7,28 +7,32 @@
  */
 
 import { v4 as uuidv4 } from 'uuid';
-import type { CrudClient } from '../../public/crud_client';
-import type { CreateIn, DeleteIn, GetIn, SearchIn, UpdateIn } from '../../common';
-
-export interface Todo {
-  id: string;
-  title: string;
-  completed: boolean;
-}
-
-export type TodoCreateIn = CreateIn<'todos', { title: string }>;
-export type TodoUpdateIn = UpdateIn<'todos', Partial<Omit<Todo, 'id'>>>;
-export type TodoDeleteIn = DeleteIn<'todos', { id: string }>;
-export type TodoGetIn = GetIn<'todos'>;
-export type TodoSearchIn = SearchIn<'todos', { filter?: 'todo' | 'completed' }>;
+import type { CrudClient } from '@kbn/content-management-plugin/public';
+import type {
+  TodoCreateIn,
+  TodoUpdateIn,
+  TodoDeleteIn,
+  TodoGetIn,
+  TodoSearchIn,
+  TodoUpdateOut,
+  TodoCreateOut,
+  TodoSearchOut,
+  TodoDeleteOut,
+  Todo,
+  TodoGetOut,
+} from '../../../../common/examples/todos';
 
+/**
+ * This client is used in the storybook examples to simulate a server-side registry client
+ * and to show how a content type can have a custom client-side CRUD client without using the server-side registry
+ */
 export class TodosClient implements CrudClient {
```

Comment:
Currently `CrudClient` is a client-side thing. 
But on the server we have ContentStorage. 

Should we align them? Both naming and function signatures?

Reply:
Not sure I understand why we need to know about storage layer at the RPC level (and align interfaces). It is an implementation details. Although it is true that teams would probably use the same interfaces for their "data" for ex.

One thing is the interfaces for the RPC layer, and another is the signature of the `Storage` class methods.